torch/onnx/symbolic_opset8.py

```diff
@@ -30,12 +30,16 @@
     Scan
 """
 
+import functools
 import warnings
 
 import torch
 from torch.onnx import _type_utils, errors, symbolic_helper, symbolic_opset9 as opset9
+from torch.onnx._internal import registration
 
-block_listed_operators = [
+_onnx_symbolic = functools.partial(registration.onnx_symbolic, opset=8)
```

Comment:
It seems that the version number is duplicated defined here. ex: We have file name telling us it's opset8, and the registry telling us again. Will we have a next step to only depend on decorator opset version number?

Reply:
Great question. Yes - with this way of registering we will not depend on how the module is named. I am looking at how we can better structure our functions. Until then the functions will stay in these versioned modules. We need to make them accessible for at least a few release versions for compatibility too. Any thoughts / suggestions?

Reply:
I guess without making each symbolic_fn as class, it only matters how we are going to categorize them. As now we don't need to categorize them in opset version way, maybe we can discuss a categorization that we can benefit from the most?

Reply:
That sounds like a great plan